setup.py

```diff
@@ -42,10 +42,10 @@ def find_scripts(script_dir, prefix):
     version=version,
     packages=find_packages(),
     install_requires=[
-        'astropy',
-        'ctapipe~=0.7.0',
+        "astropy>=3,<5",
```

Comment:
Why not astropy 4?

Reply:
I copied it from the ctapipe setup, I'm not sure if astropy 4 was _required_ by ctapipe or it was just _compatible_. I would prefer to leave it as it is in the ctapipe installation. In any case, I just checked and with this condition, the version I got is `4.0.1.post1`